### PR TITLE
[scheduler] Do not update postsubmit bringup targets in check runs

### PR DESCRIPTION
**Pull request by CaseyHillers**
https://github.com/flutter/flutter/issues/120331

### CODE REVIEW

File: app_dart/lib/src/request_handlers/postsubmit_luci_subscription.dart [hunk 1]
```diff
@@ -52,22 +51,6 @@ class PostsubmitLuciSubscription extends SubscriptionHandler {
       return Body.empty;
     }
 
-    if (buildPushMessage.userData.containsKey('repo_owner') && buildPushMessage.userData.containsKey('repo_name')) {
```
Comment (drewroengoogle) [inline app_dart/lib/src/request_handlers/postsubmit_luci_subscription.dart:55]: For my understanding, could you explain what the removal of this section does?

Comment (CaseyHillers) [reply app_dart/lib/src/request_handlers/postsubmit_luci_subscription.dart:55]: This seems to be a duplicate check of below. We already have the commit slug below via commit_key.

**Comment by drewroengoogle** [comment]
Besides the one question, LGTM!

**Review by drewroengoogle — APPROVED** [review]

**Review by ricardoamador — APPROVED** [review]
LGTM

**Pull request merged**